webknossos/tests/conftest.py

```diff
@@ -62,13 +86,135 @@ def vcr_config() -> Dict[str, Any]:
     }
 
 
+_REPLACE_IN_REQUEST_CONTENT = {
+    r"\d{4}-\d{2}-\d{2}_\d{2}-\d{2}-\d{2}": "2000-01-01_00-00-00",
+    r"\"uploadId\": \"[^\"]+\"": '"uploadId": "2000-01-01T00-00-00__0011"',
+}
+
+_REPLACE_IN_REQUEST_MULTIFORM = {
+    r"\r\n": "\n",
+    r"--[0-9a-f]+": "--",
+    r"\d{4}-\d{2}-\d{2}T\d{2}-\d{2}-\d{2}__[0-9a-f\-]+": "2000-01-01T00-00-00__0011",
+}
+
+
+def _before_record_request(request: VcrRequest) -> VcrRequest:
```

Comment:
please add comment that this is to create a readable (and relevant) diff even if the requests contain irrelevant changing fields and binary blobs